app/Http/Controllers/ProjectController.php

```diff
@@ -56,25 +57,27 @@ public function store(Request $request) {
 			]
 		]);
 
-		$project = $this->saveRecord($attributes);
+		$project = $this->saveRecord($attributes, $request->file('projectImage'));
```

Comment:
@anakarenbm  duad: por qué le pasas aqu&iacute; el file al `saveRecord`?

Reply:
emm.. true!

lo puedo sacar de attributes, ahorita lo cambio

Reply:
Ah no, ya vi que `saveRecord` es tuyo el me&acute;todo. I'm fine with it as it is if you want

Reply:
kewl